### PR TITLE
utilities and list

### PR DESCRIPTION
**Pull request by preethasri**
created text utilities and list
Text Utilities:
  headings
  small text
  gray text
  center text
Lists
  Spaced list
  Stacked list
live link: https://deploy-preview-23--ornateui.netlify.app/textUtilities/textUtilities.html
live link:https://deploy-preview-23--ornateui.netlify.app/list/list.html

### CODE REVIEW

File: list/list.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+
+/* spaced list */
+
+.spaced-list-items {
+    list-style: none;
+    margin-bottom: 1.1rem;
+    padding: 1rem 1rem;
+    text-align: center;
+    background-color: #f1f1f1;
```
Comment (Chakravarthi0) [inline list/list.css:46]: You are using the same background color for stacked-list-item and nested-list as well, I would suggest you to use a variable.

File: list/list.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+.stacked-list-items:hover {
+    background-color: coral;
+}
+
+
+/* nested list */
+
+.nested-list {
+    background-color: #f1f1f1;
+    padding: 60px 60px;
```
Comment (Chakravarthi0) [inline list/list.css:74]: It is recommended to rem.

**Review by Chakravarthi0 — APPROVED** [review]
Hi Preetha,
I have two suggestions,
-Use variables for colors(not just for hex codes, but also for names like "red", "royalblue",etc). 
-Always use rem, unless the value is less than 10px.

Rest looks good.

File: list/list.css (hunk 1 above)
Comment (preethasri) [reply list/list.css:46]: > You are using the same background color for stacked-list-item and nested-list as well, I would suggest you to use a variable.

sure saran will work on it

File: list/list.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+            <embed src="https://carbon.now.sh/embed?bg=rgba%28189%2C16%2C224%2C1%29&t=vscode&wt=none&l=htmlmixed&width=608.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=15.5px&lh=177%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cul%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253EPrice%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ECategory%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ERatings%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%2520class%253D%2522spaced-list-items%2522%253ESort%2520products%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Ful%253E"
+                style="width: 608px; height: 222px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin" type="">
```
Comment (tanveertkd) [inline list/list.html:37]: Try keeping the styling in a separate file. Here, you can try and move the iFrame styling to one of the common stylesheets.

File: list/list.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+.nested-list li {
+    list-style: none;
+    font-weight: bold;
+    margin-left: 20px;
+}
+
+.code-snippet {
+    display: flex;
+    justify-content: center;
+    margin: 20px 0px;
```
Comment (tanveertkd) [inline list/list.css:87]: Try using rems as much as you can. Could you please convert these to rems as well?

**Review by tanveertkd — APPROVED** [review]
Just those two suggestions. Try using rems wherever you can. Apart from that this is good to go.

File: list/list.css (hunk 4 above)
Comment (preethasri) [reply list/list.css:87]: will work on this tanveer . thank you

**Pull request merged**